utils/pod/util.go

```diff
@@ -115,3 +116,14 @@ func EndStep(stepData map[string][]*types.StepData, stepName string, tag map[str
 		step.Status = "Success"
 	}
 }
+
+func UpdateHealthCheckStatus(stepData map[string][]*types.StepData) {
+	for stepName, stepVals := range stepData {
+		if strings.HasPrefix(stepName, "HealthCheck-") &&
+			len(stepVals) > 0 &&
+			stepVals[len(stepVals)-1].Status == "Starting" {
```

Comment:
How about unhealthy?

Reply:
step metrics is not healthcheck specific. We have 3 step status - Starting, success, error, hence I kept the same